<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileImportDialog</class>
 <widget class="QDialog" name="FileImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File Import...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Filename:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="fileName"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectFileName">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Par File:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="fileName2"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectFileName2">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="perceiveBonds">
     <property name="toolTip">
      <string>Attempt to perceive bonds?</string>
     </property>
     <property name="text">
      <string>Perceive bonds?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileName</tabstop>
  <tabstop>selectFileName</tabstop>
  <tabstop>perceiveBonds</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
